plugins/pyembed/CMakeLists.txt

```diff
@@ -37,13 +37,14 @@ if(PY2EMBED)
     unset(PYTHON_DEBUG_LIBRARIES CACHE)
     unset(PYTHONLIBS_VERSION_STRING CACHE)
     ADD_PLUGIN(py2embed PACKAGES PythonLibs MINVERSION 2.6 MAXVERSION 2.7)
-    set(PYTHON2LIBS_FOUND ${PYTHONLIBS_FOUND})
-    set(PYTHON2_LIBRARY ${PYTHON_LIBRARY})
-    set(PYTHON2_LIBRARIES ${PYTHON_LIBRARIES})
-    set(PYTHON2_INCLUDE_PATH ${PYTHON_INCLUDE_PATH})
-    set(PYTHON2_INCLUDE_DIR ${PYTHON_INCLUDE_DIR})
-    set(PYTHON2_DEBUG_LIBRARIES ${PYTHON_DEBUG_LIBRARIES})
-    set(PYTHON2LIBS_VERSION_STRING ${PYTHONLIBS_VERSION_STRING})
+
+    set(PYTHON2LIBS_FOUND ${PYTHONLIBS_FOUND} CACHE BOOL "Found Python2 Libraries")
+    set(PYTHON2_LIBRARY ${PYTHON_LIBRARY} CACHE PATH "Library")
+    set(PYTHON2_LIBRARIES ${PYTHON_LIBRARIES} CACHE PATH "Libraries")
+    set(PYTHON2_INCLUDE_PATH ${PYTHON_INCLUDE_PATH} CACHE PATH "include path")
+    set(PYTHON2_INCLUDE_DIR ${PYTHON_INCLUDE_DIR} CACHE PATH "include dir")
+    set(PYTHON2_DEBUG_LIBRARIES ${PYTHON_DEBUG_LIBRARIES} CACHE PATH "include debug libraries")
+    set(PYTHON2LIBS_VERSION_STRING ${PYTHONLIBS_VERSION_STRING} CACHE STRING "version")
```

Comment:
Any reason some of the string labels are different here than above ?
("Library" vs "library" for example)
("include debug libraries" vs "debug libraries for example")